src/components/Proposals/ProposalDetails/VoteActions.js

```diff
@@ -0,0 +1,251 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Button,
+  GU,
+  IconCheck,
+  IconConnect,
+  IconCross,
+  Info,
+  noop,
+  RADIUS,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import {
+  DISPUTABLE_VOTE_STATUSES,
+  VOTE_STATUS_SCHEDULED,
+} from '../disputable-vote-statuses'
+import { dateFormat, toMs } from '../../../utils/date-utils'
+import { useWallet } from '../../../providers/Wallet'
+
+const VoteActions = React.memo(
+  ({ vote, onVoteYes, onVoteNo, onExecute, onChangeVote }) => {
+    const theme = useTheme()
+    const { account } = useWallet()
+    const disputableStatus = DISPUTABLE_VOTE_STATUSES.get(vote.status)
+    const { snapshotBlock, startDate, hasEnded, voterInfo, orgToken } = vote
+
+    if (disputableStatus !== VOTE_STATUS_SCHEDULED) {
+      return <></>
+    }
+
+    if (!account || !voterInfo) {
```

Comment:
I think just checking for the account would be enough for this block

Reply:
Is not enough cause first the account is connected and then the promises for voterInfo get executed. And if voterInfo is still undefined, it breaks